app/canopy-app.html

```diff
@@ -338,9 +338,9 @@
       <div class="modal-body">
         <p>Choose format...</p>
         <paper-menu>
-          <paper-item>WAV (RIFF) 16-bit PCM</paper-item>
+          <paper-item disabled>WAV (RIFF) 16-bit PCM</paper-item>
```

Comment:
This should not be disabled because user might want to get the truncated 16-bit version.